src/layout/DetailedTimer.tsx

```diff
@@ -25,6 +25,10 @@ export default class DetailedTimer extends React.Component<Props> {
         }
 
         const totalHeight = this.props.state.timer.height + this.props.state.segment_timer.height;
+        const topRowHeight = totalHeight * 0.55;
+        const bottomRowHeight = totalHeight - topRowHeight;
+        const topRowFontSize = topRowHeight * 0.5 * 0.85;
+        const bottomRowFontSize = bottomRowHeight * 0.5 * 0.8 * 0.85;
```

Comment:
These 0.85 and 0.8 seem a bit magical here. Those are the same as the ones in variables.scss right? I guess they should be named constants in there then.

Reply:
Oh wait, I guess importing from scss to js is probably not possible right? Oof

Reply:
Apparently this works with webpack: https://github.com/LiveSplit/LiveSplitOne/blob/2ad32229541b75187409759379eddf6e1f3f534f/src/css/variables.scss#L35-L39

Pretty sweet 😄